Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -127,8 +129,8 @@ bool JitArm64::HandleFault(uintptr_t access_address, SContext* ctx)
       {
         ERROR_LOG_FMT(DYNA_REC,
                       "JitArm64 address calculation overflowed. This should never happen! "
-                      "PC {:#018x}, access address {:#018x}, memory base {:#018x}, MSR.DR {}",
-                      ctx->CTX_PC, access_address, memory_base, m_ppc_state.msr.DR);
+                      "PC {:#018x}, access address {:#018x}, memory base {:#018x}, MSR.DR {}, memptr {:#018x}, pbase {:#018x}, lbase {:#018x}",
+                      ctx->CTX_PC, access_address, memory_base, m_ppc_state.msr.DR, (u64)m_ppc_state.mem_ptr, (u64)memory.GetPhysicalBase(), (u64)memory.GetLogicalBase());
```

Comment:
We prefer C++ style casts (`static_cast`) over C-style casts.

Actually, there's something even better you can do here: You can use `fmt::ptr`. Example: https://github.com/dolphin-emu/dolphin/blob/7f40c6f2f86cb3f77b155240df0e512dfe1eef29/Source/Core/Common/JitRegister.cpp#L108